types/pull-stream/sources/error.d.ts

```diff
@@ -3,5 +3,5 @@ import pull = require('..');
 /**
  * Create a stream with no contents (it just ends immediately).
  */
-declare function error(): pull.Source<never>;
+declare function error(err: Error): pull.Source<never>;
```

Comment:
This doesn't actually have to be an `Error` object, it could be anything or even `undefined`. I agree that it should be used with `Error` objects, but there's nothing in the code that forces you to do so.

Reply:
Here's a bug, I meant to use `error(err: pull.EndOrError)`.